mkdir: cannot create directory ‘inputs/1e54’: File exists
Traceback (most recent call last):
  File "scripts/get_centopo.py", line 26, in <module>
    if topos[cen]=='IN':
KeyError: '37'
mkdir: cannot create directory ‘inputs/1qd6’: File exists
Traceback (most recent call last):
  File "scripts/get_centopo.py", line 26, in <module>
    if topos[cen]=='IN':
KeyError: '38'
mkdir: cannot create directory ‘inputs/3v8x’: File exists
Traceback (most recent call last):
  File "scripts/get_centopo.py", line 26, in <module>
    if topos[cen]=='IN':
KeyError: '315'
mkdir: cannot create directory ‘inputs/4q35’: File exists
Traceback (most recent call last):
  File "scripts/get_centopo.py", line 26, in <module>
    if topos[cen]=='IN':
KeyError: '245'
